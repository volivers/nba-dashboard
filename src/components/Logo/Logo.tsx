import React from 'react';

export interface LogoProps {
  name?: string;
}

const Logo = ({ name }: LogoProps) => (
  <img className="mx-2" src={`img/${name}.svg`} alt={`${name}-logo`} height="40" loading="lazy" />
);

export default Logo;