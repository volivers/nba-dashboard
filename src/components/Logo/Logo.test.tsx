import { render } from '@testing-library/react';
import Logo, { LogoProps } from './Logo';

describe('Logo', () => {
  let props: LogoProps;

  beforeEach(() => {
    props = { name: 'Test' };
  });

  it('renders properly', () => {
    const { container } = render(<Logo {...props} />);

    expect(container).toMatchSnapshot();
  });
});
