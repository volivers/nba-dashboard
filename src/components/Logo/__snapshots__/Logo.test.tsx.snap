// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders properly 1`] = `
<div>
  <img
    alt="Test-logo"
    class="mx-2"
    height="40"
    loading="lazy"
    src="img/Test.svg"
  />
</div>
`;
